fe/src/main/java/org/apache/doris/analysis/SetVar.java

```diff
@@ -128,6 +127,17 @@ else if (result instanceof IntLiteral) {
                 result = new IntLiteral(SqlModeHelper.encode(sqlMode).toString(), Type.BIGINT);
             }
         }
+        if (variable.equalsIgnoreCase(GlobalVariable.HEARTBEAT_FLAGS)) {
```

Comment:
The way of analyzing variables has been changed.
You'd better wait this PR merged and change it to that way:
https://github.com/apache/incubator-doris/pull/2355

Reply:
It seems that it is not necessary to be that complicated, I use a easier realization.